src/webaudio.js

```diff
@@ -275,6 +275,27 @@ export default class WebAudio extends util.Observer {
         this.gainNode.connect(this.ac.destination);
     }
 
+    /**
+     * Set the sink id for the media player
+     *
+     * @param {string} deviceId String value representing audio device id.
+     */
+    setSinkId(deviceId) {
+        if (deviceId) {
+            let audio = new window.Audio();
+            if (!audio.setSinkId) {
+                console.warn('setSinkId is not supported in your browser')
+                return;
+            }
+            audio.autoplay = true;
+            var dest = this.ac.createMediaStreamDestination();
+            this.gainNode.disconnect();
+            this.gainNode.connect(dest);
+            audio.src = URL.createObjectURL(dest.stream);
+            audio.setSinkId(deviceId);
+        }
+    }
```

Comment:
@johnryan why are you creating a new Audio instance here? Can the regular one not be (re-)used?

Reply:
@thijstriemstra which instance are you referring to? The webaudio apis don't support the `setSinkId` concept so I create an HTMLAudioElement and attach the stream to that.

Reply:
I see: https://stackoverflow.com/a/43069474

```
Unfortunately, setting the destination audio device of a webaudio graph isn't yet implemented, and the api for this is not yet finalized.

What you can do for now, is connect the webaudio graph to an HTML element, and set the sinkid of the element (currently works on Chrome only)
```

Can you add a summary of that in a multiline comment so it's clear why you're doing that?

Reply:
And I would also throw errors instead of showing the warning, console.log is not used inside the library itself.

Reply:
@thijstriemstra done

Reply:
Awesome, thanks.